contracts/OrganizationInterface.sol

```diff
@@ -0,0 +1,108 @@
+pragma solidity ^0.4.23;
+
+// Copyright 2018 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+/**
+ *  @title OrganizationInterface
+ *
+ *  @notice Provides organization interface
+ */
+contract OrganizationInterface {
```

Comment:
We should probably start using the `interface` type: https://solidity.readthedocs.io/en/develop/contracts.html#interfaces

Reply:
This updates isWorker to be external which is public now as interface keyword bring warning "Warning: Functions in interfaces should be declared external."

Reply:
I think that while the interface has to be `external`, the implementation can be `public`, but I could be mistaken. Please verify.